### ISSUE TITLE
How to define custom mode

### ISSUE THREAD

**Issue by KeqiZeng** [DCsunset/vscode-modal-editor#12]
Hi,

thanks for this good plugin. But I have a maybe stupid question: how can we define a custom mode? I can't find the answer neither in the README nor in the issues. Maybe I missed something?

**Comment by DCsunset** [comment]
Yes, you can. As said in the README,  you are free to add more modes. You can do this just by giving it a name and declaring its keybindings like other predefined modes.
